content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1006,6 +1006,63 @@ status:
   terminating: 3 # three Pods are terminating and have not yet reached the Failed phase
 ```
 
+### Success policy {#success-policy}
+
+{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+
+{{< note >}}
+You can only configure a success policy for an Indexed Job if you have the
+`JobSuccessPolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+enabled in your cluster.
+{{< /note >}}
+
+When you run an indexed Job, a success policy defined with the `spec.successPolicy` field, 
```

Comment:
```suggestion
When you run an Indexed Job, a success policy defined with the `spec.successPolicy` field, 
```

Reply:
Also, try to be more direct:
When creating an Indexed Job, you can define when a Job can be declared as succeeded using a `successPolicy`, based on the pods that succeeded.

Reply:
It sounds good to me.

Reply:
Done.